alb-lambda-rest-api-sam-py/example-pattern.json

```diff
@@ -0,0 +1,76 @@
+{
+  "title": "AWS Lambda REST API and Amazon ALB with Listener Rules",
+  "description": "This pattern creates an Application Load Balancer with route-based listener rules, paired with an AWS Lambda RESTful API function as the target. It uses Python 3.9 and the Serverless Application Model (SAM) CLI.",
```

Comment:
The description has a maximum length of 150 characters, please shorten it a bit, for example

> This pattern creates an Application Load Balancer with route-based listener rules, paired with an AWS Lambda function as the target.

as the usage of SAM and Python is expressed via the `language` and `framework` fields already